plugins/modules/ec2_instance_info.py

```diff
@@ -36,6 +36,15 @@
     required: false
     aliases: ['uptime']
     type: int
+  include_attributes:
+    description:
+      - Describes the specified attributes of the returned instances. You can specify only one attribute at a time.
+        See U(https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstanceAttribute.html) for possible values.
+    required: false
+    type: list
+    elements: str
+    aliases: ['attributes']
```

Comment:
Can you also please list the supported attributes as choices?

Reply:
done